public/app.scss

```diff
@@ -44,3 +47,10 @@ $euiTextColor: $euiColorDarkestShade !default;
 .state-accordion:hover {
   text-decoration: none;
 }
+
+.euiFilterGroup__popoverPanel {
+  width: fit-content;
```

Comment:
Does max-width alone not result in the element stretching as required for the content?

Reply:
it does not.  the Eui component has it's own defaults which seem to render that ineffective.